examples/job/get-file-from-job.py

```diff
@@ -0,0 +1,282 @@
+# 
+# 
+# ## Steps
+# 
+# In this notebook, we will accomplish the following:
+# 
+# 1. Import the structure of cubic Ge from Materials Project
+# 2. Set up and run a single-point calculation using VASP
+# 3. List the files currently in the job's directory
+# 4. Get download links for the OUTCAR and CONTCAR files of a VASP run
+# 5. Decode the OUTCAR file into UTF-8, and write the last few contents to the screen
+# 6. Save both the OUTCAR and CONTCAR files to disk.
+# 
+# ## Pre-requisites
+# 
+# The explanation below assumes that the reader is familiar with the concepts used in Exabyte platform and RESTful API. We outline these below and direct the reader to the original sources of information:
+# 
+# - [Generating RESTful API authentication parameters](../system/get_authentication_params.ipynb)
+# - [Importing materials from materials project](../material/import_materials_from_materialsproject.ipynb)
+# - [Creating and submitting jobs](../job/create_and_submit_job.ipynb)
+
+# ## Execution
+# 
+# 
+# ### Import packages
+
+# In[]:
+
+
+from IPython.display import JSON, display
+import os
+import sys
+import urllib
+
+# Import settings file and utils file
+module_path = os.path.abspath(os.path.join('..'))
+if module_path not in sys.path: sys.path.append(module_path)
+from settings import ENDPOINT_ARGS, ACCOUNT_ID, MATERIALS_PROJECT_API_KEY
+from utils import wait_for_jobs_to_finish, get_property_by_subworkow_and_unit_indicies, dataframe_to_html, ensure_packages_are_installed
+ensure_packages_are_installed()
+
+# Relevant functions from the API client
+from exabyte_api_client.endpoints.jobs import JobEndpoints
+from exabyte_api_client.endpoints.projects import ProjectEndpoints
+from exabyte_api_client.endpoints.materials import MaterialEndpoints
+from exabyte_api_client.endpoints.bank_workflows import BankWorkflowEndpoints
+from exabyte_api_client.endpoints.raw_properties import RawPropertiesEndpoints
+
+
+# #### Materials
+#     
+# - **MATERIALS_PROJECT_IDS**: a list of material IDs to be imported from materials project
+# - **TAGS**: a list of [tags](https://docs.exabyte.io/entities-general/data/#tags) to assign to imported materials
+# - **MATERIALS_SET_NAME**: the name of the materials set
+# 
+
+# In[]:
+
+
+MATERIALS_PROJECT_ID = ["mp-32"]
+MATERIALS_SET_NAME = "materials-set"
+TAGS = ["tag1", "tag2"]
+
+
+# #### Jobs
+# 
+# Parameters for the jobs to be ran for the imported materials:
+# 
+# - **JOB_NAME_PREFIX**: prefix to be used for the job name with "{JOB_NAME_PREFIX} {FORMULA}" convention (e.g.  "Job Name Prefix - SiGe")
+# - **JOBS_SET_NAME**: the name of the jobs set
+
+# In[]:
+
+
+JOB_NAME_PREFIX = "Ge_Test"
+JOBS_SET_NAME = "jobs-set"
+
+
+# #### Workflow
+# 
+# This example is based on [this](https://platform.exabyte.io/analytics/workflows/SnGspffD7nPsBHJaQ) bank workflow which is later copied to the account workflows collection.
+
+# In[]:
+
+
+BANK_WORKFLOW_ID = "SnGspffD7nPsBHJaQ"
```

Comment:
(1) the workflow needs a little more explanation, as we faced earlier with run-job example, let's use a similar pattern, please, https://github.com/Exabyte-io/exabyte-api-examples/pull/12/commits/33f2da4365e5b3a530f6c96ace53f558ae905fa3
(2) let's use QE for this example, this would help everyone run it

Reply:
The QE run takes just under 5 minutes to run. Might be worth making a demo workflow just for this.

Maybe a gold unit cell with LDA (which gives surprisingly good Au bond distances compared to other more expensive functionals) and a fairly light K-mesh?

Reply:
Switching to Si sped things up by quite a bit